content/doc/book/platform-information/upgrade-java-to-11.adoc

```diff
@@ -0,0 +1,79 @@
+. Upgrade the required plugins (see <<Upgrading Plugins>>)
+
+Starting with the Jenkins 2.357 weekly release and the Long Term Support release 2.361.1, Java 11 or Java 17 is required.
+
+== Upgrading Plugins
+
+It's important not just to upgrade Jenkins and the JVM, but also to upgrade the plugins which support Java 11.
+Plugin upgrades assure compatibility with the most recent Jenkins releases.
+
+NOTE: If you discover a previously unreported issue, please let us know: read link:/participate/report-issue/#issue-reporting[how to report an issue] for guidance.
+
+// Commented because pipeline support plugin 3.0 is over 3 years old and has 8+ later releases
+//
+// One of the most important plugin upgrades is the plugin:workflow-support[Pipeline: Support plugin]: make sure that the version of the plugin is at least `3.0`.
+//
+// NOTE: Stop all Pipeline jobs before upgrading this plugin because this upgrade changes the serialization of Pipeline builds. As a general rule, even though Pipeline jobs are supposed to survive a Jenkins restart, it's always a better option to make sure that no Pipeline builds are in progress before any scheduled Jenkins maintenance.
+
+== Jakarta XML Binding
+
+Some plugins use JAXB libraries provided by the JDK.
+However, the `java.xml.bind` and `javax.activation` modules are no longer included in OpenJDK 11, and plugins might fail if no replacement is offered.
+
+To fix this problem, we've bundled those libraries into a new detached plugin: plugin:jaxb[JAXB plugin].
+When any Jenkins core more recent than `2.163` is running on Java 11, this plugin is automatically installed.
+However, if you manage your plugins outside Jenkins, for example, if you use `plugins.txt` in your Docker images, you might need to install the plugin explicitly.
+
+== JVM version on agents
+
+All agents must be running on the same JVM version as the controller due to how controllers and agents communicate.
+If you're upgrading your Jenkins controller to run on Java 11, you also need to upgrade the JVM on your agents.
+
+You can validate the version of each agent with the plugin:versioncolumn[Versions Node Monitors] plugin.
+This plugin provides information about the JVM version of each agent on the node management screen of your Jenkins instance.
+You can also configure this plugin to automatically disconnect any agent with an incorrect JVM version.
+
+== Java Web Start
+
+Java Web Start has been removed in Java 11.
+When a Jenkins controller runs on Java 11, the Java Web Start button will no longer appear in the Web UI.
+You can't launch agents for a Java 11 Jenkins controller from a `*.jnlp` file downloaded to a web browser.
+
+There are no plans to replace this functionality.
+Connect agents to Jenkins on Java 11 with plugins like plugin:ssh-slaves[SSH Build Agents Plugin], with operating system command line calls to `java -jar agent.jar`, or using containers.
+
+== JDK Tool Automatic installer
```

Comment:
```suggestion
== JDK Tool Automatic installer
```
Not sure if the capitalization here is correct. I couldn't find a reference to it.

Reply:
I found a reference, it does seem to be the name of the tool https://learn.microsoft.com/en-us/azure/devops/pipelines/tasks/reference/java-tool-installer-v0?view=azure-pipelines